### PR TITLE
Feat(PP-60): updated placeholder texts

### PR DESCRIPTION
**Pull request by DmitrijSidorkin**
Will update the high scores texts in `account-main` page once I complete another task (Add high score categories based on session length in user model.(PP-65)).

Changing placeholder texts in detailed results is now listed as a separate task in Jira as it requires changing how game data is stored, so that it can be properly displayed later on.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -8,11 +8,8 @@ const flash = require("connect-flash");
 const passport = require("passport");
 const LocalStrategy = require("passport-local");
 const MongoStore = require("connect-mongo");
-<<<<<<< HEAD
 const bodyParser = require("body-parser");
-=======
 const favicon = require("serve-favicon");
->>>>>>> 27428c9 (feat(PP-59): added favicon)
```
Comment (elnino97) [inline index.js:15]: Leftover from rebasing

File: views/account/account-main.ejs [hunk 2]
```diff
@@ -3,29 +3,63 @@
...
+    <div class="info-box-div">
+      <h3 class="highscore-title">High Scores</h3>
+      <div class="grid-container">
+        <div class="grid-item"></div>
+        <div class="grid-item gird-difficulty">
+          <p class="grid-text-full">Very Easy</p>
+          <p class="grid-text-short">V.Easy</p>
+        </div>
+
+        <div class="grid-item gird-difficulty">Easy</div>
```
Comment (elnino97) [inline views/account/account-main.ejs:23]: What is `gird`? Seems like a typo

File: views/account/account-main.ejs [hunk 3]
```diff
@@ -3,29 +3,63 @@
...
+        <div class="grid-item grid-score">score</div>
+        <div class="grid-item grid-score">score</div>
+        <div class="grid-item grid-score">score</div>
+        <div class="grid-item grid-score">score</div>
+        <div class="grid-item grid-length">5</div>
+        <div class="grid-item grid-score">score</div>
+        <div class="grid-item grid-score">score</div>
+        <div class="grid-item grid-score">score</div>
+        <div class="grid-item grid-score">score</div>
+        <div class="grid-item grid-score">score</div>
```
Comment (elnino97) [inline views/account/account-main.ejs:60]: `grid-item` is not targeted by any css selectors so no point in bloating HTML with unused classes.

File: views/account/account-main.ejs [hunk 4]
```diff
@@ -3,29 +3,63 @@
...
-      stats: Lorem
+    <div class="info-box-div account-first-row">
+      <div class="personal-info">
+        <p class="personal-info-item">Name:</p>
+        <p class="personal-info-item">Age:</p>
+        <p class="personal-info-item">Location:</p>
+      </div>
+      <p class="bio">bio</p>
+    </div>
+    <div class="info-box-div">
```
Comment (elnino97) [inline views/account/account-main.ejs:14]: Something I noticed before and in some cases didn't comment but I would prefer if you kept the tag name (div) out of the class name. In HTML it's already obvious that it's a div, and if it is not in while reading your CSS spreadsheet, then a more descriptive class name should be used. In that case I would prefer `personal-info-wrapper`.

File: index.js (hunk 1 above)
Comment (DmitrijSidorkin) [reply index.js:15]: yes

File: views/account/account-main.ejs [hunk 5]
```diff
@@ -3,61 +3,61 @@
...
       </div>
       <p class="bio">bio</p>
     </div>
-    <div class="info-box-div">
+    <div class="info-box-wrapper">
       <h3 class="highscore-title">High Scores</h3>
       <div class="grid-container">
-        <div class="grid-item"></div>
-        <div class="grid-item gird-difficulty">
+        <div class=""></div>
```
Comment (elnino97) [inline views/account/account-main.ejs:17]: Empty class name

**Review by elnino97 — APPROVED** [review]

**Pull request merged**